ttkwidgets/font/propertiesframe.py

```diff
@@ -2,18 +2,47 @@
 Author: RedFantom
 License: GNU GPLv3
 Source: This repository
+
+Edited by rdbende: translation and default font properties option
 """
 # Based on an idea by Nelson Brochado (https://www.github.com/nbro/tkinter-kit)
 import tkinter as tk
 from tkinter import ttk
+from locale import getdefaultlocale
+
+# --- Translation
+EN = {"Font properties:": "Font properties:"}
+
+FR = {"Font properties:": "Propriétés"}
```

Comment:
Propriétés works, but if we were to be specific, it would be `Propriétés de police d'écriture` or just `Propriétés de police`.

It depends on the context (source : am french)